test/functional/mempool_expiry.py

```diff
@@ -26,73 +26,78 @@
+
+        # Broadcast another (independent) transaction.
+        independent_txid = self.wallet.send_self_transfer(from_node=node, utxo_to_spend=independent_utxo)['txid']
 
-        # Let most of the timeout elapse and check that the parent tx is still
-        # in the mempool.
-        nearly_expiry_time = entry_time + 60 * 60 * timeout - 5
+        # Let most of the timeout elapse and check that the parent tx is still in the mempool.
+        nearly_expiry_time = entry_time + 3600 * timeout - 5
         node.setmocktime(nearly_expiry_time)
-        # Expiry of mempool transactions is only checked when a new transaction
-        # is added to the to the mempool.
-        node.sendtoaddress(node.getnewaddress(), 1.0)
+        self.trigger_mempool_expiry()
         self.log.info('Test parent tx not expired after {} hours.'.format(
-            timedelta(seconds=(nearly_expiry_time-entry_time))))
+            timedelta(seconds=(nearly_expiry_time - entry_time))))
         assert_equal(entry_time, node.getmempoolentry(parent_txid)['time'])
 
-        # Transaction should be evicted from the mempool after the expiry time
-        # has passed.
-        expiry_time = entry_time + 60 * 60 * timeout + 5
+        # Transaction should be evicted from the mempool after the expiry time has passed.
+        expiry_time = entry_time + 3600 * timeout + 5
         node.setmocktime(expiry_time)
-        # Expiry of mempool transactions is only checked when a new transaction
-        # is added to the to the mempool.
-        node.sendtoaddress(node.getnewaddress(), 1.0)
+        self.trigger_mempool_expiry()
         self.log.info('Test parent tx expiry after {} hours.'.format(
-            timedelta(seconds=(expiry_time-entry_time))))
-        assert_raises_rpc_error(-5, 'Transaction not in mempool',
-                                node.getmempoolentry, parent_txid)
+            timedelta(seconds=(expiry_time - entry_time))))
+        assert_raises_rpc_error(-5, 'Transaction not in mempool', node.getmempoolentry, parent_txid)
 
         # The child transaction should be removed from the mempool as well.
         self.log.info('Test child tx is evicted as well.')
-        assert_raises_rpc_error(-5, 'Transaction not in mempool',
-                                node.getmempoolentry, child_txid)
+        assert_raises_rpc_error(-5, 'Transaction not in mempool', node.getmempoolentry, child_txid)
+
+        # Check that the other tx is still in the mempool.
+        self.log.info('Test other tx not expired after {} hours.'.format(
+            timedelta(seconds=(expiry_time - half_expiry_time))))
+        assert_equal(half_expiry_time, node.getmempoolentry(independent_txid)['time'])
 
     def run_test(self):
-        self.log.info('Test default mempool expiry timeout of %d hours.' %
-                      DEFAULT_MEMPOOL_EXPIRY)
+        self.log.info('Test default mempool expiry timeout of {} hours.'.format(DEFAULT_MEMPOOL_EXPIRY))
```

Comment:
not sure about mixing all the formatting changes in with the other changes. In either case if you want to change this, you can use f-strings